src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html

```diff
@@ -1,14 +1,9 @@
 <article>
     <header class="l-page-header">
         <h3 class="l-page-header__title">
-            Available Catalogs
+            Available Catalogs <br>
+            <small class="text-muted">Select a catalog to manage category level assignments.</small>
         </h3>
-        <!--====== CREATE ======
-        <div class="l-page-header__btns">
-            <button type="button" class="btn btn-primary btn-block" ng-click="buyerCatalogs.createCatalog()"><i class="fa fa-plus-circle"></i>
-                New Catalog
-            </button>
-        </div>-->
```

Comment:
I believe that diff is telling you I already removed it (it's highlighted in red)